designdocs/passkey_p256.md

```diff
@@ -105,6 +117,44 @@ These are the following technical terms and specifications for the implementatio
 - `passkey_sig_pk` : Signature generated by Passkey on  `account_pk`
 - `account_sig_passkey_pk` : Signature generated by Account Key on `passkey_pk`
 - `passkey_tx_signature` : Signature generated by Passkey on transaction payload, this is required for transaction submission.
+- `passkey_authenticator_data` : Authenticator data generated by Passkey, this is required for transaction submission.
+- `passkey_client_data` : Client data generated by Passkey, this is required for transaction submission.
+- `nonce` : A unique number used to prevent replay attacks.
+
+### Passkey Transaction Payload
+
+- The transaction payload is a structure that contains the following fields:
+  
+    ```typescript
+
+    interface PasskeyPayload {
+      passkeyPublicKey: `passkey_pk`;
+      passkeySignature: `passkey_tx_signature`;
+      passkeyAuthenticator: `passkey_authenticator_data`;
+      passkeyClientDataJson: `passkey_client_data`;
+      passkeyCall: PasskeyCall;
+      nonce: `nonce`;
```

Comment:
I think nonce should be inside `PasskeyCall` since we would need to sign it to remove replay attacks

Reply:
If we try to add mortality it should also be inside the PasskeyCall